src/components/select/Select.js

```diff
@@ -97,6 +97,10 @@ export class SelectComponent extends BaseComponent {
     if (typeof data === 'string') {
       return this.t(data);
     }
+    if (!data.data) {
+      return '' || this.component.placeholder
```

Comment:
Isn't this the same thing as just saying... ```return this.component.placeholder```?

And on that note, why in the world would we ever return the component placeholder as the "value" of the component. If the value is empty, we should see that it is empty in all displays. Having it instead show ```Select your Country``` as the value would be pretty strange.

Reply:
Also, what is ```data.data``` here?  Looking at the code below this, I am not sure how this is protecting anything since ```data.data``` is not referenced anywhere here. Maybe the code should be

```
getView(data) {
  return this.itemTemplate({data});
}
```
Not sure, but this is something that a good breakpoint would easily show you what is going on.